api-catalog-package/src/main/resources/bin/start.sh

```diff
@@ -187,6 +188,15 @@ if [ $JAVA_VERSION -ge 61 ]; then
     fi
 fi
 
+if [ "${ATTLS_ENABLED}" = "true" ]; then
+  keystore_type=
+  keystore_pass=
+  key_pass=
+  key_alias=
+  keystore_location=
```

Comment:
This could be just personal preference 
```suggestion
  unset keystore_type
  unset keystore_pass
  unset key_pass
  unset key_alias
  unset keystore_location
```